### PR TITLE
Fix/write workflows

### PR DESCRIPTION
**Pull request by andre-merzky**
The current implementation of `write_workflow` only writes the last workflow.  This fix ensures that all submitted workflows are recorded for a session.

### CODE REVIEW

File: src/radical/entk/utils/prof_utils.py [hunk 1]
```diff
@@ -219,42 +220,53 @@ def get_session_description(sid, src=None):
     return desc
 
 
-def write_workflow(workflow, uid, workflow_fout='entk_workflow', fwrite=True):
+# ------------------------------------------------------------------------------
+#
+def write_workflow(workflows, uid):
+def write_workflow(workflows, uid, workflow_fout='entk_workflow', fwrite=True):
```
Comment (iparask) [inline src/radical/entk/utils/prof_utils.py]: @andre-merzky did you happen to define twice the same function?

Comment (andre-merzky) [reply src/radical/entk/utils/prof_utils.py]: Thanks - fixed!1

**Comment by andre-merzky** [comment]
@iparask : I'm afraid this needs a new review, will go into the next release.

**Comment by andre-merzky** [comment]
@iparask : This should be ready for a new review.

**Pull request merged**